### ISSUE TITLE
Error when entering to an app

### ISSUE THREAD

**Issue by Exatoon27** [srcbookdev/srcbook#437]
## Problem

When I try to enter to an app, the program crashes with a git error, but I can't understand what is exactly the problem. I searched it in google, and it seems to be relative to git commit, and I don't understand why is there a git commit.

### Screenshot
![Captura de pantalla 2024-11-11 014751](https://github.com/user-attachments/assets/880edac0-e972-4268-9b3a-891d40c3d7e2)

## Environment details

I used the `npx srcbook@latest start` and `npm install -g srcbook` and no one worked, so I don't think that it's the problem. I'm in Windows 11 and I have git installed and I only configured the name and email.

##

I don't know if it's necessary, but when I say app I refer to this:

![Captura de pantalla 2024-11-11 015735](https://github.com/user-attachments/assets/7f5cb67b-eda5-45fe-896d-fa48e590ee84)
##

So, can someone explain to me what's the error?

**Comment by nichochar** [comment]
This bug should be fixed in 0.0.15. 

Try:
- deleting the app
- install the latest version (0.0.15) with `npx srcbook@latest start`
- create a new app

If this still doesn't work, let me know

**Comment by Exatoon27** [comment]
Same behavior and now is using the 0.0.15
![Captura de pantalla 2024-11-11 213107](https://github.com/user-attachments/assets/2bcdc1f7-0184-48ce-b971-fd6531988c4a)

**Comment by nichochar** [comment]
Yeah, we don't support windows yet unfortunately, as described in https://github.com/srcbookdev/srcbook/pull/444

To solve this, we brought the app online at https://srcbook.com

It's currently free to use, let me know if this solves your needs!
